2_Growth-Maturity-Decline.md

```diff
@@ -3,75 +3,26 @@
 
  Activity Metric | Description
  --- | ---
- Tickets Closed | Total number of tickets closed
- Time to Close | Time from bug opened to bug closed
- Time to Merge | Time from submission to merge or abandon
- Review Efficiency | Number of merged patches / Number of abandoned patches
+ Tickets Closed / Tickets Opened | Total number of tickets closed per tickets opened over time

```

Comment:
This is, for any given period (eg, a month), the number of tickets closed during that period (whenever they were opened) by the number of tickets opened during that month, right?

Reply:
Yes, the time period can be specified by the user of the system.